src/bilder/images/airbyte/airbyte.pkr.hcl

```diff
@@ -0,0 +1,92 @@
+locals {
+  timestamp = regex_replace(timestamp(), "[- TZ:]", "")
+  app_name  = "airbyte"
+}
+
+variable "build_environment" {
+  type        = string
+  default     = "operations-ci"
+}
+
+variable "business_unit" {
+  type        = string
+  default     = "operations"
+}
+
+# Available options are "web" or "worker". Used to determine which type of node to build an image for.
+variable "node_type" {
+  type    = string
+  default = "server"
+}
+
+source "amazon-ebs" "airbyte" {
+  ami_description         = "Deployment image for Airbyte server generated at ${local.timestamp}"
+  ami_name                = "airbyte-${var.node_type}-${local.timestamp}"
+  ami_virtualization_type = "hvm"
+  instance_type           = "t3a.large"
+  launch_block_device_mappings {
+    device_name           = "/dev/sda1"
+    volume_size           = 25
+    delete_on_termination = true
+  }
+  run_tags = {
+    Name    = "${local.app_name}-${var.node_type}-packer-builder"
+    OU      = "${var.business_unit}"
+    app     = "${local.app_name}"
+    purpose = "${local.app_name}-${var.node_type}"
+  }
+  run_volume_tags = {
+    Name    = "${local.app_name}-${var.node_type}-packer-builder"
+    OU      = "${var.business_unit}"
+    app     = "${local.app_name}"
+    purpose = "${local.app_name}-${var.node_type}"
+  }
+  snapshot_tags = {
+    Name           = "${local.app_name}-${var.node_type}-ami"
+    OU             = "${var.business_unit}"
+    app            = "${local.app_name}"
+    purpose        = "${local.app_name}-${var.node_type}"
+  }
+  # Base all builds off of the most recent docker_baseline_ami built by us, based of Debian 11
+  source_ami_filter {
+    filters = {
+      name                = "docker_baseline_ami-${var.node_type}-*"
```

Comment:
Is this the motivation for creating a new Packer template?

Reply:
Yes.